sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/ms-teams.yaml

```
@@ -13,7 +13,7 @@ provider:
       # - https://alelmsa.webhook.office.com/webhookb2/xxxx-xxxx-xxx-xxx@xxxxxxxx-xxxx-xxxx-xxxxxxx/IncomingWebhook/xxxx/xx
       # - https://test.webhook.office.com/webhookb2/11111111-4444-4444-8444-cccccccccccc@22222222-4444-4444-8444-cccccccccccc/IncomingWebhook/33333333012222222222333333333344/44444444-4444-4444-8444-cccccccccccc
       patternNot: 
-        "(?i)\
+        - "(?i)\
         test|\
         abcd|\
         1234|\
```

```suggestion
        - "(\\w)\\1{5,}"
        - "(?i)test|abcd|1234"
        - "(?i)\
```
Can you remove the ` (\\w)\\1{5,}|\` just under that please ?